GunShipAlpha/src/Player.java

```diff
@@ -0,0 +1,45 @@
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import javax.imageio.ImageIO;
+import javax.swing.*;
+
+public class Player {
+    private int health = 10;
+    private BufferedImage sprite;
+    public Player(){
+        loadSprite();
+    }
+    void loadSprite() {
+        try {
+            sprite = ImageIO.read(new File("src/Images/GunShip.png"));
+        } catch (IOException e) {
+            throw new RuntimeException("Failed to load sprite", e);
+        }
+    }
+
+    public void reduceHealth(){
+        System.out.println("reducing health");
+        health--;
+        isDead();
```

Comment:
no point to calling isDead since it just returns a boolean

Reply:
fixed